### DISCUSSION TITLE
Set id for altInput config

### DISCUSSION THREAD

**Discussion by ikimes** [ankurk91/vue-flatpickr-component#290] [Q&A]
Is there a way to set an id attribute for the hidden alt input when altInput: true? Our QA uses Katalon to automate and it would be helpful to provide a unique id to the flatpickr instance, one for the visible input and another for the hidden altInput. Been looking around the docs and not seeing a way to accomplish this.

Thanks.

**Comment by mark1502** [comment]
Also wondering the same thing, because I can't seem to set focus to a flat-pickr component which uses altInput.
